Source/ARTPresenceMap.m

```diff
@@ -175,10 +173,10 @@ - (void)startSync {
 
 - (void)endSync {
     ARTLogVerbose(_logger, @"%p PresenceMap sync ending", self);
-    [self cleanUpAbsentMembers];
```

Comment:
Can we undo this PR's changes to `endSync`, per https://github.com/ably/ably-cocoa/pull/1714#issuecomment-1857699353?

Reply:
https://github.com/ably/ably-cocoa/pull/1714/commits/2de35dbf55a1ff531a915889f64725dcb660d003